README.md

```diff
@@ -51,6 +51,132 @@ After this, you will have a directory containing files used for creating a
 custom Jupyter widget. You will now be able to easily package and distribute
 your custom Jupyter widget.
 
+## Files Generated by cookiecutter
+Jupyter widgets are made of two parts:
```

Comment:
Maybe:

Jupyter widgets are made of two parts -> Jupyter widgets are made of two separate programs

?